instance/models/openedx_appserver.py

```diff
@@ -142,21 +149,42 @@ def set_field_defaults(self):
         # assert that it isn't set because if a ValidationError occurred, this method could be
         # called multiple times before this AppServer is successfully created.
         self.configuration_settings = self.create_configuration_settings()
+        self.lms_user_settings = self.create_lms_user_settings()
         super().set_field_defaults()
 
+    def default_playbook(self):
+        """
+        Return a Playbook instance for the standard configuration playbook.
+        """
+        return Playbook(
+            source_repo=self.configuration_source_repo_url,
+            requirements_path='requirements.txt',
+            playbook_path='playbooks/{}.yml'.format(self.RUN_ROLE_PLAYBOOK),
+            version=self.configuration_version,
+            variables=self.configuration_settings,
+        )
+
+    def lms_user_creation_playbook(self):
+        """
+        Return a Playbook instance for creating LMS users.
+        """
+        return Playbook(
+            source_repo=self.configuration_source_repo_url,
+            requirements_path='requirements.txt',
```

Comment:
@smarnach Before running the playbooks, the code in ansible.py creates a new python virtualenv and installs ansible and everything that's listed in `requirements.txt`. By separation, I mean that if we run multiple playbooks, it's possible that they come from different repos or different versions of the same repo, so the contents of requirements.txt could differ. Thus, it's important in theory that each time we run a playbook we provision a separate virtualenv.

In practice, the temporary virtualenv created for `CONFIGURATION_PLAYBOOK` here is going to be identical to the one created for `RUN_ROLE_PLAYBOOK`